### PR TITLE
3d text rendering

### PR DESCRIPTION
**Pull request by tpaviot**
This branch adds an Addons module, where we'll put anything that can't go somewhere else.

The naming used in this PR is not consistent with CamelCase. I'm not absolutely sure about that, going with `display_available_fonts` or `DisplayAvailableFonts`

### CODE REVIEW

File: examples/core_font_3d_console.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        InteractiveConsole.runcode(self, codez)
+        sys.stdout = sys.__stdout__
+        sys.stderr = sys.__stderr__
+        return out.getvalue()
+
+c = Interpreter()
+
+# black background
+display.set_bg_gradient_color(0, 0, 0, 0, 0, 0)
+display.FitAll()
```
Comment (jf---) [inline examples/core_font_3d_console.py]: [amazing example](https://www.youtube.com/watch?v=A9UW5pk38GY), so funny! you had me fooled ;)

**Comment by jf---** [comment]
@tpaviot , this feature seems well developed / complete / passes CI...
curious to know if there is anything stopping you from merging?
really great that pyocc has got a new primitive 🤗

**Comment by tpaviot** [comment]
@jf--- I was waiting for a +1 before merging, if you're ok we can go

**Comment by jf---** [comment]
@tpaviot , thanks for that! All looks perfect to me 👍

**Pull request merged**